resources/views/families/members/_form.blade.php

```diff
@@ -0,0 +1,12 @@
+<?php
+    $selectedOption = [];
+?>
+<div class="form-group">
+    <label>{{ __('organisations.fields.members') }}</label>
+
+    <select multiple="multiple" name="members[]" id="members" class="form-control form-members" style="width: 100%" data-url="{{ route('characters.find', ['with_family' => '1']) }}">
+        @foreach ($selectedOption as $key => $val)
```

Comment:
Not sure what $selectedOption is or where it's from. Seems like this can be skipped?